<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>865</width>
    <height>761</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>company</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>40</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>window_size</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="window_size">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>40</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="train_button">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>40</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>train</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>40</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateEdit" name="start_date_predict">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>450</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>450</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>starting_date</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>40</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>end_date</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="end_date_train">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>40</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>48</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;train&gt;</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="loss_plot">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>811</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>loss plot</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;predict&gt;</string>
    </property>
   </widget>
   <widget class="QToolButton" name="browse_model">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>450</y>
      <width>24</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QComboBox" name="models">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>450</y>
      <width>321</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>450</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>load_model</string>
    </property>
   </widget>
   <widget class="QPushButton" name="predict">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>450</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>predict</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>40</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>start_date</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="start_date_train">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>40</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QProgressBar" name="train_pbar">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>120</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>log</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="log_text">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>120</y>
      <width>571</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="prediction_plot">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>490</y>
      <width>811</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QDialogButtonBox" name="buttonBox">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>690</y>
      <width>156</width>
      <height>24</height>
     </rect>
    </property>
    <property name="standardButtons">
     <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>70</y>
      <width>48</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>epochs</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="epochs">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>70</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>70</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>learning_rate</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="learning_rate">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>70</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>input_dim</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>70</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>hidden_dim</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>70</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>num_layers</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>70</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>output_dim</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="input_dim">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>70</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="output_dim">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>70</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="hidden_dim">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>70</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="num_layers">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>70</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>865</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFiles">
    <property name="title">
     <string>Files</string>
    </property>
    <addaction name="actionClose"/>
   </widget>
   <addaction name="menuFiles"/>
  </widget>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionopen">
   <property name="text">
    <string>open</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
